src/components/forgot-password-drawer/ForgotPasswordDrawer.tsx

```diff
@@ -42,7 +42,7 @@ const createStyles = (themeColors: ThemeColors) =>
     },
 
     restoreAccount: {
-      fontWeight: '700',
+      fontWeight: '600',
```

Comment:
I'm pretty sure fontWeight has no effect on Android, would be good to double check

Reply:
Converted to use weight on the text property instead and works on both android & ios!